scripts/fast-reboot

```diff
@@ -110,6 +110,10 @@ function clear_warm_boot()
     if [[ -f ${WARM_DIR}/${REDIS_FILE} ]]; then
         mv -f ${WARM_DIR}/${REDIS_FILE} ${WARM_DIR}/${REDIS_FILE}.${TIMESTAMP} || /bin/true
     fi
+    # enabling swss auto restart
+    docker exec swss supervisorctl start supervisor-proc-exit-listener || /bin/true
+    sed -i -e "s/\<RestartSec=300\>/RestartSec=30/" /etc/systemd/system/swss.service || /bin/true
+    systemctl daemon-reload
```

Comment:
this is not needed because we only call it from where no trap would be invoked?

Reply:
Removed this as now I have moved the disable-swss-autorestart after `+e` and removing trap (right before killing swss),